lib/src/entrypoint.dart

```diff
@@ -246,13 +246,15 @@ class Entrypoint {
 
     if (dryRun) {
       result.summarizeChanges(type, dryRun: dryRun);
+      result.showOutdated(type);
```

Comment:
Ah, sorry I didn't explain this well enough.

I meant to change `SolveResult.summarizeChanges` like this:

```dart
  /// Displays a one-line message summarizing what changes were made (or would
  /// be made) to the lockfile.
  ///
  /// If [type] is `SolveType.UPGRADE` it also shows the number of
  /// packages that are not at the latest available version.
  ///
  /// [type] is the type of version resolution that was run.
  void summarizeChanges(SolveType type, {bool dryRun = false}) {
    final report = SolveReport(type, _sources, _root, _previousLockFile, this);
    report.summarize(dryRun: dryRun);
    if (type == SolveType.UPGRADE) {
      report.reportOutdated();
    }
  }
```

Reply:
Ohh, sorry!

Updated the code.